frappe/desk/form/meta.py

```diff
@@ -198,7 +198,12 @@ def set_translations(self, lang):
 	def load_dashboard(self):
 		if self.custom:
 			return
-		self.set('__dashboard', self.get_dashboard_data())
+
+		dashboard_data = self.get_dashboard_data()
+		for hook in frappe.get_hooks("override_doctype_dashboards", {}).get(self.name, []):
```

Comment:
This logic should be moved to `get_dashboard_data` method

Reply:
Fixed